docs/source/index.rst

```diff
@@ -32,49 +31,54 @@ You can easily deploy Sycamore locally or on a virtual machine using Docker.
 
 With Docker installed:
 
-
-
-1. Clone the Sycamore repo:
+1.	Clone the Sycamore repo: 
 
 ``git clone https://github.com/aryn-ai/sycamore``
 
-2. Set OpenAI Key:
+2.	Set OpenAI Key:
 
 ``export OPENAI_API_KEY=YOUR-KEY``
 
-3. Go to:
+3.	Go to: 
 
 ``/sycamore``
 
-4. Launch Sycamore. Containers will be pulled from DockerHub:
+4.	Launch Sycamore. Conatainers will be pulled from DockerHub:
 
-``docker compose up --pull=always``
+``Docker compose up --pull=always``
 
+5.	The Sycamore demo query UI will be at localhost:3000
 
-**Use the service:**
+You can next choose to run a demo that `prepares and ingests data from the Sort Benchmark website<docs/source/welcome_to_sycamore/get_started.md#demo-ingest-and-query-sort-benchmark-dataset>`_, [crawl data from a public website](docs/source/welcome_to_sycamore/get_started.md#demo-ingest-and-query-data-from-an-arbitrary-website), or write your own data preparation script.
```

Comment:
I think single backtick will render inline in command font.  Is that what you want for `prepares and ingests data...`?  It feels weird to me as I expect to only see commands/code in `this format`

Reply:
I was trying to get the link to work.